src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -95,6 +96,21 @@ public static InputStream openInputStream(String path)
         return inputStream;
     }
 
+    public static InputStream openInputStream(String path, Function<SeekableByteChannel, SeekableByteChannel> wrapper)
```

Comment:
Docs for this method, including mention of under what circumstances the wrapper is applied